apps/trading-e2e/src/fixtures/markets/active/chart.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
comment (non-blocking): not TS or anything so if the API changes and these files go out of date we won't be notified.

Reply:
converted to generator functions colocated with queries